coderd/coderd.go

```diff
@@ -399,8 +399,14 @@ func New(options *Options) *API {
 					r.Get("/roles", api.userRoles)
 
 					r.Route("/keys", func(r chi.Router) {
-						r.Post("/", api.postAPIKey)
```

Comment:
This route was used on the FE for CLI auth. We'll have to change it there too. Otherwise, this will break it.

Reply:
I did not find anything calling this route from our api.ts or codersdk, do you know of somewhere else?